docs/.vuepress/components/UpdatedHomePage.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="pa-1 l-up-pl-2 l-up-pr-2">
+    <div class="flex flex-column">
+      <div class="flex space-between align-center">
+        <h1 class="l5 max-w-55ch">
+          Welcome to Ethereum, a community-driven internet built for you
```

Comment:
@jjmstark should confirm but I think to begin with, we want to keep the copy the same (i.e. `translateString('page-home-title')`) or "Ethereum is a global, open-source platform for decentralized applications."

Reply:
Yes, headline copy for english should remain the same for now. As discussed in Figma thread, we *should* update this but I want to disentangle that update from shipping the new homepage design. 

(but as noted, we can make content adjustments as needed before shipping - these initial PRs from @carlfairclough are just about getting the design elements finalized)